cli/text_encoding.rs

```diff
@@ -54,6 +54,30 @@ pub fn strip_bom(text: &str) -> &str {
   }
 }
 
+static SOURCE_MAP_PREFIX: &str =
+  "//# sourceMappingURL=data:application/json;base64,";
+
+pub fn source_map_from_code(code: &str) -> Option<Vec<u8>> {
```

Comment:
Is there context for these?  How and where are they used.  Do they respect the compiler options?

Reply:
This was an existing function that I extracted it out to here for reuse, but the other one is new. This function would mostly only be used now if someone ran a JS file with a source map embedded in it and not for an emitted files except when running with `--inspect`, in which case it would be embedded (so source maps work in inspector), which is what the other function is used for.

Could you clarify the question about respecting compiler options?